generator/src/main/java/com/scottlogic/deg/generator/restrictions/RowSpec.java

```diff
@@ -31,6 +31,8 @@ public ProfileFields getFields() {
         return fields;
     }
 
+    public Map<Field, FieldSpec> getFieldToFieldSpec() { return fieldToFieldSpec; }
```

Comment:
Why has this been introduced?

Reply:
To support tests. We may be able to remove this by adding an equals() for the type